.gitignore

```diff
@@ -96,6 +96,9 @@ samples/client/petstore/silex/SwaggerServer/venodr/
 samples/server/petstore/php-symfony/SymfonyBundle-php/Tests/cache/
 samples/server/petstore/php-symfony/SymfonyBundle-php/Tests/logs/
 
+#PHP-laravel
+samples/server/petstore/php-laravel/node_modules
+samples/server/petstore/php-laravel/vendor
```

Comment:
This line is unnecessary as 'vendor' is ignored [here](https://github.com/ackintosh/openapi-generator-1/blob/44f34cc569859b43cf0de52d367d833b7281d03f/.gitignore#L92).